Lib/fontTools/merge.py

```diff
@@ -780,6 +849,7 @@ def merge(self, fontfiles):
 		# TODO Is it necessary to reload font?  I think it is.  At least
 		# it's safer, in case tables were loaded to provide glyph names.
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		self.fonts = copy.deepcopy(fonts)
```

Comment:
I'm drunk.. It was originally added to make test isGlyphSame easier. Since we haven't write test case for isGlphSame yet. I'm removing the deepcopy here.

The self.fonts = fonts is added to the code. We isGlyphSame need to access self.fonts since it need to read glyphs from fonts and compare them.